ratatoskr/static/scss/style.scss

```diff
@@ -3,4 +3,13 @@
 @import "app/var-override.scss";
 @import "bootstrap/bootstrap.scss";
 // Append style overrides after Bootstrap
-@import "app/link-disable.scss";
\ No newline at end of file
+@import "app/link-disable.scss";
+
+.card-hover-shadow {
```

Comment:
Instead of putting styles directly into the `style.scss`, you should make a new Sass file, like `app/card.scss`, and put all card related styles there, then import the file here using an `@import` directive. The `@import` directive effectively pastes the contents of the file. 

The reason why we prefer splitting up styles into multiple files is to avoid having one comically large stylesheet that is hard to read through.

Reply:
Though the hover effect is definitely an improvement over the previous nothing we had